### ISSUE TITLE
Request: Make history searchable

### ISSUE THREAD

**Issue by lotsahelp** [bambulab/Bambu-Handy#181]
We print several different models and searching through history to reprint would be a nice addition. Right now, for items that are a couple pages down in the history, it's quicker and easier to just reprint from Studio. Please give us a way to search history to make this faster.

Thank you

**Comment by jebsimmons** [comment]
@lotsahelp Agreed. The date feature is nice, but being able to search Print History by name would solve a lot of scrolling.
